data/intro missions.txt

```diff
@@ -61,7 +61,7 @@ mission "Intro [0]"
 			`	"Great," he says. "My name is James, by the way."`
 			`	"<first> <last>," you say.`
 			`	"I'm looking forward to traveling with you, Captain <last>," says James.`
-			`	As you're helping him wheel his luggage aboard and showing him his bunk, he says, "Before we take off, you might head on over to the trading center and stock up on medical goods; we can sell those for a good profit on <planet>. Or take a look at the outfitter if you like, but they don't sell much here compared to on the big manufacturing worlds."`
+			`	As you're helping him wheel his luggage aboard and showing him his bunk, he says, "Before we take off, you might head on over to the trading center and stock up on medical goods; we can sell those for a good profit on <planet>.  Also make sure to visit the local outfitter, while they don't sell much here compared to on the big manufacturing worlds, every world with an outfitter will at least carry a map with information on the neighboring systems, I'd suggest buying one so we don't get lost.`
```

Comment:
```suggestion
			`	As you're helping him wheel his luggage aboard and showing him his bunk, he says, "Before we take off, you might head on over to the trading center and stock up on medical goods; we can sell those for a good profit on <planet>.  Also make sure to visit the local outfitter, while they don't sell much here compared to on the big manufacturing worlds, every world with an outfitter will at least carry a map with information on the neighboring systems, I'd suggest buying one as it takes a bit of the guesswork out of navigation. It isn't a requirement, though: Some captains like exploring instead."`
```

I don't like the "I'd suggest buying one so we don't get lost." as it makes it sounds almost like a requirement. I would prefer it if this was framed as a helpful suggestion instead.

Reply:
I'm not sure about it seeming like a requirement. It seems to me like "I'd suggest buying one so we don't get lost." is already being framed as a helpful suggestion. 

James isn't telling you to buy one before he'll continue working with you, he's just suggesting you do so in order to prevent getting lost and I'm not sure it really needs to be explicitly stated to be "not a requirement".

I also want to encourage players to purchase a map at least once, rather than make it seem like an optional choice they 
can make if they choose not to engage in exploration. Once they've purchased their first map, they can use that knowledge to make an informed decision as to whether they want to purchase more. 

I'm absolutely open to changing it though, if you do think it would be best, I just think it would be good to strongly encourage a new player to buy a map _at least once_

Reply:
Honestly, I *don't* think they should buy one their first time through. Didn't the first couple playthroughs of EV, EVO, and EVN, either.

At the very least, they shouldn't be buying maps until after they've made a few jumps and explored a little. They should actually experience the intended normal exploration first so they actually understand what the map does and how it changes their game.

Reply:
Okay, understood. If you feel that is best, I will make the change.